app/assets/stylesheets/_tables.scss

```diff
@@ -0,0 +1,36 @@
+[aria-sort] > * {
+  position: relative;
+
+  &:after {
+    font-size: em(10, 19);
+    position: absolute;
+    right: -15px;
+    top: 50%;
+    margin-top: -5px;
+  }
+
+  &:visited {
+    color: $link-colour;
+  }
+
+  &:hover {
+    color: $link-hover-colour;
+  }
+}
+
+[aria-sort=descending],
+[aria-sort=ascending] {
+    padding-right: em(30, 19);
+}
+
+[aria-sort=descending] > * {
+  &:after {
+    content: '\00a0\00a0\25B2';
```

Comment:
The two spaces (to the left of the arrow) don't seem to be necessary because the arrow is absolutely positioned to the right.
As it's not obvious what the characters are, I would add a comment, saying that it is an "up (or down) arrow/triangle".